tests/effects/toutparam.nim

```diff
@@ -2,7 +2,7 @@ discard """
   cmd: '''nim c --warningAsError:Uninit:on --skipCfg --skipParentCfg $file'''
   errormsg: "use explicit initialization of 'x' for clarity [Uninit]"
   line: 24
-  disabled: "true"
+    disabled: "true"
```

Comment:
created disabled around 2020-06 in a commit that references 14521

		Running it gives:

			Expected:
			use explicit initialization of 'x' for clarity [Uninit]

			Gotten:
			use explicit initialization of 'tmp' for clarity [Uninit]

Reply:
Do we have a issue number?

Reply:
no, but a textual description is fine.

I want to be careful before we commit to this being a "for real" known issue, because as we're working through the spec things are still settling, so something like this we'll end up revisiting.

Reply:
How should a textual description be done? in the known issue field?

Reply:
Yeah, I would put the text in the known issue field.